scripts/acceptOwnership.js

```diff
@@ -0,0 +1,49 @@
+// ES5 style
+const ConnexDriver = require('@vechain/connex-driver');
+const Framework = require("@vechain/connex-framework").Framework;
+const find = require('lodash/find');
+
+const config = require('./deploymentConfig');
+const Multirewards = require('../build/contracts/MultiRewards.json');
+
+const { Driver, SimpleNet, SimpleWallet } = ConnexDriver;
+
+const [network, multiRewards] = process.argv.slice(2)
+
+// ensure we have appropriate arguments
+if (!network || !multiRewards) {
+  console.error("Usage: node scripts/acceptOwnership [mainnet|testnet] [Multirewards address]");
+
+  process.exit();
+} else if (network ==='mainnet') {
+  const input = readlineSync.question("Confirm you want to execute this on the MAINNET? (y/n) ");
+
+  if (input != 'y') process.exit(1);
+}
```

Comment:
If possible, let's have this prompt after the "Using wallet address:" line so that the executor can see which wallet address they're using before executing it